sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeHealthcareEntitiesTests.cs

```diff
@@ -170,12 +168,14 @@ public async Task RecognizeHealthcareEntitiesBatchWithErrorTest()
 
             await operation.WaitForCompletionAsync(PollingInterval);
 
-            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+            AnalyzeHealthcareEntitiesResultCollection resultCollection = operation.Value;
 
             Assert.IsNotNull(resultCollection[2].Id);
-            Assert.IsNotNull(resultCollection[2].TextAnalyticsError);
-            Assert.AreEqual("Document text is empty.", resultCollection[2].TextAnalyticsError.Message);
-            Assert.AreEqual(TextAnalyticsErrorCode.InvalidDocument, resultCollection[2].TextAnalyticsError.ErrorCode.ToString());
+
+            var exceptionMessage = "Document text is empty.";
+            Assert.IsTrue(resultCollection[2].HasError);
+            Assert.AreEqual(exceptionMessage, resultCollection[2].Error.Message);
```

Comment:
in the SDK we usually don't check the `Message` as the service can decide to change it (it is not a breaking change) and by doing so they will break our tests.